src/pip/_internal/commands/cache.py

```diff
@@ -66,6 +69,13 @@ def run(self, options, args):
 
         return SUCCESS
 
+    def get_cache_dir(self, options, args):
+        # type: (Values, List[Any]) -> None
+        if args:
+            raise CommandError('Too many arguments')
```

Comment:
A test hitting this branch would be great! :)

Reply:
👍 Test added in 6b640d1254ef9f39aaf443c242d6bbe80d8e0f35